### PR TITLE
CAT-2696 Refactor LegoNXTSensorConfigInfoDialog and LegoEV3SensorConf…

### PR DESCRIPTION
**Pull request by LPeteR90**
…igInfoDialog

-Merge two Dialogs containing very similar functionality
-Refactor methods in ProjectActivity that display the dialog
-Refactor some names in ProjectActivity & SettingsFragment to be clearer about purpose
-Remove unnecessary member var from ProjectManager.loadProject()

### CODE REVIEW

**Comment by rschluesselbauer** [comment]
Looks good to me.

File: catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java [hunk 1]
```diff
@@ -192,40 +190,18 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 		}
 	}
 
-	private boolean needToShowLegoEV3InfoDialog() {
-		boolean isLegoEV3InfoDialogDisabled = SettingsFragment
-				.getShowLegoEV3MindstormsSensorInfoDialog(getApplicationContext());
+	private void showLegoSensorConfigInfo() {
+		int requiredResources = ProjectManager.getInstance().getCurrentProject().getRequiredResources();
+		boolean nxtDialogEnabled = SettingsFragment.getLegoNXTSensorInfoDialogEnabled(this);
```
Comment (LPeteR90) [inline catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java]: remove getLegoNXTSensorInfoDialogEnabled from SettingsFragment and get directly from sharedprefs

File: catroid/src/main/java/org/catrobat/catroid/ui/dialogs/LegoSensorConfigInfoDialog.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+			titleStringResId = R.string.lego_ev3_sensor_config_info_title;
+			infoStringResId = R.string.lego_ev3_sensor_config_info_text;
+			sensorMapping = SettingsFragment.getLegoEV3SensorMapping(getActivity());
+			sensorMappingStrings = getResources().getStringArray(R.array.ev3_sensor_chooser);
+		} else {
+			throw new IllegalArgumentException("LegoSensorConfigInfoDialog: Constructor called with invalid sensor type");
+		}
+
+		View dialogView = View.inflate(getActivity(), R.layout.dialog_lego_sensor_config_info, null);
```
Comment (LPeteR90) [inline catroid/src/main/java/org/catrobat/catroid/ui/dialogs/LegoSensorConfigInfoDialog.java]: add @dimen/material_spacing_large to layout

File: catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java (hunk 1 above)
Comment (LPeteR90) [reply catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java]: done - note: some settings regarding lego are still in the SettingsFragment. Removing that probably should be a separate ticket

File: catroid/src/main/java/org/catrobat/catroid/ui/dialogs/LegoSensorConfigInfoDialog.java (hunk 2 above)
Comment (LPeteR90) [reply catroid/src/main/java/org/catrobat/catroid/ui/dialogs/LegoSensorConfigInfoDialog.java]: done

**Comment by thmq** [comment]
please run espresso tests

**Comment by LPeteR90** [comment]
please run espresso tests

**Pull request merged**